### DISCUSSION TITLE
Roots model

### DISCUSSION THREAD

**Discussion by Inferdy** [mxgmn/MarkovJunior#29] [Show and tell]
Guys, try this one. It looks like roots/branches/foliage/swarm and it's so simple. Sadly, it contains conflicts between concurrent convertations, but I like the result.

model:
```xml
<all in="*I*" out="IWI" values="AIW" origin="Center"/>
```
ruleset:
```xml
<model name="Roots" size="60" steps="100" gif="True"/>
```

![Roots](https://user-images.githubusercontent.com/24939153/173920698-b0436adf-748e-41f9-823f-db18472adfde.gif)
![33](https://user-images.githubusercontent.com/24939153/173920752-1b66d824-d8fe-4d3d-827e-76645dbf929f.png)
![43](https://user-images.githubusercontent.com/24939153/173920773-c00a5d67-bdbe-447a-9e00-4ed84cf3aed7.png)
![80](https://user-images.githubusercontent.com/24939153/173920788-308a058e-9ddf-412e-a754-086605a1a20e.png)

**Comment by mxgmn** [comment]
Cool! You don't need to create a Markov node if it's only one node inside. This is equivalent to
```xml
<all in="*I*" out="IWI" values="AIW" origin="True"/>
```

**Reply by Inferdy** [reply]
```xml
<prl values="AIW" origin="True" in="*I*" out="IWI"/>
```
leads to
```
Roots > unknown origin name True
```
But I saw some of your models have origin="True" aswell.

**Reply by mxgmn** [reply]
I [updated](https://github.com/mxgmn/MarkovJunior/commit/7dfc1b7611abc4b0236f38f84adbec9087a04df4) origin to be boolean, because I don't really need random origin. But the release hasn't been updated yet.
